From: Patrick Menschel <menschel.p@posteo.de>
Date: Wed, 28 Oct 2020 20:35:35 +0100
Subject: Add mcp251xfd driver as module to RPI build configuration

This Patch adds MCP251XFD=m  Kconfig flag to the 32 bit RPI defconfig files:

bcmrpi_defconfig: Pi 1, Pi Zero, Pi Zero W
bcm2709_defconfig: Pi 2, Pi 3, Pi 3+, Pi CM3
bcm2711_defconfig: Pi 4

...and the 64 RPI defconfig files.
bcmrpi3_defconfig: Pi 3, Pi 3+, Pi CM3
bcm2711_defconfig: Pi 4

Signed-off-by: Patrick Menschel <menschel.p@posteo.de>
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index b062fead409b..096894330856 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -416,6 +416,7 @@ CONFIG_CAN=m
 CONFIG_CAN_VCAN=m
 CONFIG_CAN_SLCAN=m
 CONFIG_CAN_MCP251X=m
+CONFIG_CAN_MCP251XFD=m
 CONFIG_CAN_EMS_USB=m
 CONFIG_CAN_GS_USB=m
 CONFIG_BT=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index 598e9e198951..1a6853beda54 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -417,6 +417,7 @@ CONFIG_CAN=m
 CONFIG_CAN_VCAN=m
 CONFIG_CAN_SLCAN=m
 CONFIG_CAN_MCP251X=m
+CONFIG_CAN_MCP251XFD=m
 CONFIG_CAN_EMS_USB=m
 CONFIG_CAN_GS_USB=m
 CONFIG_BT=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 520d0bf8c700..bd00983d72a9 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -410,6 +410,7 @@ CONFIG_CAN=m
 CONFIG_CAN_VCAN=m
 CONFIG_CAN_SLCAN=m
 CONFIG_CAN_MCP251X=m
+CONFIG_CAN_MCP251XFD=m
 CONFIG_CAN_EMS_USB=m
 CONFIG_CAN_GS_USB=m
 CONFIG_BT=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 39f2fc51de4d..3de6d6dd9ed3 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -416,6 +416,7 @@ CONFIG_CAN=m
 CONFIG_CAN_VCAN=m
 CONFIG_CAN_SLCAN=m
 CONFIG_CAN_MCP251X=m
+CONFIG_CAN_MCP251XFD=m
 CONFIG_CAN_EMS_USB=m
 CONFIG_CAN_GS_USB=m
 CONFIG_BT=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 3d43b416bce4..3c2980cc8f35 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -407,6 +407,7 @@ CONFIG_YAM=m
 CONFIG_CAN=m
 CONFIG_CAN_VCAN=m
 CONFIG_CAN_MCP251X=m
+CONFIG_CAN_MCP251XFD=m
 CONFIG_CAN_EMS_USB=m
 CONFIG_BT=m
 CONFIG_BT_RFCOMM=m
